Sources/Workspace/UserToolchain.swift

```diff
@@ -19,6 +19,11 @@ private let whichArgs: [String] = ["xcrun", "--find"]
 #else
 private let whichArgs = ["which"]
 #endif
+#if os(Windows)
+private let executableExt = ".exe"
```

Comment:
I'd call it `executableSuffix`. "extension" typically excludes the '.' separator.

Reply:
Also, don't compute this at compile-time. Instead, do a runtime check using `destination.target.os == .windows`

Reply:
This is looking for executables to run locally on the host system, so the extension should always match the compiler time check, right?

Reply:
Ah, sorry, you're right. Please call it `hostExecutableSuffix` to make that clear.

Often in build systems, people will check the OS at compile time in places where it should be a run-time check (to allow for cross compilation), so it's a bit of a reflex for me :)

Reply:
`hostExecutableSuffix` it is then! Definitely better safe than sorry on checking host vs target.